charts/backbeat/templates/lifecycle/conductor_deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat.fullname" . }}-lifecycle-conductor
+  labels:
+    app: {{ template "backbeat.name" . }}-lifecycle
+    chart: {{ template "backbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.lifecycle.conductor.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-lifecycle
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-lifecycle-conductor
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/src/app/docker-entrypoint.sh"]
+          args: ["npm", "run", "lifecycle_conductor"]
+          env:
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-quorum:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: KAFKA_HOSTS
+              value: "{{- printf "%s-kafka-0.%s-kafka:9092" .Release.Name .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: MONGODB_HOSTS
+              value: "{{ .Release.Name }}-mongodb-replicaset-0.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-1.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-2.{{ .Release.Name }}-mongodb-replicaset:27017"
+            - name: AUTH_TYPE
+              value: service
+            - name: AUTH_ACCOUNT
+              value: service-lifecycle
+            - name: EXTENSIONS_LIFECYCLE_ZOOKEEPER_PATH
+              value: "{{ .Values.lifecycle.zookeeper.path }}"
+            - name: EXTENSIONS_LIFECYCLE_BACKLOG_METRICS_ZKPATH
+              value: "{{ .Values.lifecycle.zookeeper.backlog_metrics.path }}"
+            - name: EXTENSIONS_LIFECYCLE_BACKLOG_METRICS_INTERVALS
+              value: "{{ .Values.lifecycle.zookeeper.backlog_metrics.interval }}"
+            - name: EXTENSIONS_LIFECYCLE_BUCKET_TASK_TOPIC
+              value: "{{ .Values.lifecycle.kafka.bucket_task_topic }}"
+            - name: EXTENSIONS_LIFECYCLE_OBJECT_TASK_TOPIC
+              value: "{{ .Values.lifecycle.kafka.object_task_topic }}"
+            - name: EXTENSIONS_LIFECYCLE_CONDUCTOR_CRONRULE
```

Comment:
If we are doing cron'y things in our solution, wouldn't it make sense to use Kubernetes' `CronJob` objects?

Reply:
I have no particular knowledge of cron jobs in kub but why not since it might be easier to control and visualize, though I think it's better to do this in a next step (more code changes to conductor process than just deployment settings).